pkg/nets/nets.go

```diff
@@ -54,5 +54,5 @@ func ConvertPortToBigEndian(little uint32) uint32 {
 
 // ConvertIpByteToUint32 converts ip to little-endian uint32 format
 func ConvertIpByteToUint32(ip []byte) uint32 {
-	return binary.LittleEndian.Uint32(ip)
+	return binary.BigEndian.Uint32(ip)
```

Comment:
please update the function comments as well

Reply:
https://github.com/kmesh-net/kmesh/blob/67e834659309957b31e63c60cfc507d61c4a4a9b/pkg/nets/nets.go#L29
Should this function also be modified?

Reply:
It depends on the caller, i think we get the IP slices from istiod.